src/briefcase-forms.rst

```diff
@@ -57,98 +101,128 @@ Pulling forms from Collect
+   If you have incomplete forms that you cannot finalize before pulling into Briefcase, delete them. If you need to keep them, make a copy :file:`/sdcard/odk` before deleting them, and restore it after you are finished.
+
+#. Create a zip archive of the entire :file:`odk` directory.
+
+   .. tip::
+   
+     You'll need to use an app for this. 
+     
+     One option is `OI File Manager <https://play.google.com/store/apps/details?id=org.openintents.filemanager>`_.
+     
+#. Connect your Android device to your computer using a USB cable and choose to mount it as a Media device.
+#. Copy the zip file you created from the Android device to your local hard drive.
+#. Once it is copied onto your local hard drive, unzip the file.
+#. In Briefcase, open the :guilabel:`Pull` tab.
+#. Select *Custom Path to ODK Directory* in the :guilabel:`pull data from` drop-down.
+#. Select the unzipped :file:`odk` folder.
+#. Click :guilabel:`Pull`.
+#. On the Android device, open ODK Collect and delete the filled-in forms.
+
+
+   .. tip::
+  
+     - You can use the *Custom path to ODK Directory* any time you want to pull forms from custom location.
+     - You can confirm that the forms have been successfully pulled into Briefcase by confirming a successful pull status or by verifying the data appearing in a :ref:`CSV export file <briefcase-export-to-csv>`.
 
-.. warning::
- - ODK Briefcase does not work with the USB-mounted Media device (MTP) protocol, which replaced the USB-mounted drive protocol on Android 4.x devices.
- - ODK Briefcase does not support pushing blank forms to ODK Collect. You can manually install the forms on your ODK Collect device.
+     
+.. note:: 
 
+  ODK Briefcase does not support pushing blank forms to ODK Collect. Instead, :ref:`manually load the forms on your ODK Collect device <loading-forms-directly>`.
 
-.. _export-forms:
+
+.. _briefcase-export-to-csv:
 
 Export forms to CSV
 ---------------------
 
-- Open *ODK Briefcase*.
-- In the window which opens up select :guilabel:`export` tab.
-- Select the form you wish to export from the :guilabel:`form` dropdown. It displays a list of all forms on your ODK Briefcase Storage.
-- Specify the location where you wish to export form as CSV on your system in the :guilabel:`Export Directory` option.
-- Click on :guilabel:`Export` at the bottom of the window.
+#. Open the :guilabel:`Export` tab.
+#. Choose an Export Location.
+#. If exporting :doc:`encrypted-forms`, identify the location of your :ref:`PEM file <create-key>`.
+#. If you wish, select Start and End dates to specify a limited date range to export.
+#. Select the forms to export.
+#. Click :guilabel:`Export`.
 
 
 .. _cli-use:
 
-Working with command line
-----------------------------
+Working with the command line
+-------------------------------
 
-In Briefcase v1.4.4 and later, there is a scriptable command line interface.
+.. versionadded:: 1.4.4
```

Comment:
Is this how we do it else? Seems kinda jarring.

Reply:
We've only used this a few places, and this is the first time I've tried it on a larger section rather than on a single paragraph. 

I think some styling (#539) would help, and I'm open to suggestions.